webapp/repos.py

```diff
@@ -57,6 +57,8 @@ def _filter_modified_since(self, repos_to_process, modified_since_dt):
         for label in repos_to_process:
             for repo_id in self.cache.repolabel2ids.get(label, []):
                 repo_detail = self.cache.repo_detail[repo_id]
+                if label in filtered_repos_to_process:
+                    continue
```

Comment:
I think the 'if label in filtered_repos_to_process:' could be moved before the repo_detail assignment statement above... if we're going to skip this label, might as well avoid doing anything else.  :)

Reply:
Sure :)